CMake/MiscFunctions.cmake

```diff
@@ -35,35 +35,34 @@ function(add_warnings_to_targets targets)
 
     if (NOT MSVC)
         set(CHECKED_WARNING_FLAGS
+          "-Wabsolute-value"
```

Comment:
To save time auditing, here's what I did:

1. Add `"-Wsign-compare"` to the end of the list.
2. Sorted the list using VS Code.
3. Removed any duplicates I noticed.